pages/docs/dev-docs/dev-naming.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Naming conventions
+keywords: pages, development, naming
+permalink: dev-docs-naming.html
+summary: This page describes naming conventions for the preCICE ecosystem.
+---
+
+## The preCICE name
+
+preCICE is always stylized with lowercase `pre` and uppercase `CICE`, even in the beginning of a sentence.
+
+In technical contexts (e.g., a repository or package name), the lowercase `precice` is typically preferred.
+
+The forms `PreCICE` or `PRECICE` should be avoided, if possible.
+
+## Naming adapters
+
+Wherever possible, adapters use composite names that start with the name of the code they adapt, followed by preCICE. For example, human-friendly names of adapters (e.g., appearing in a publication) are `FEniCS-preCICE` and `OpenFOAM-preCICE`.
+
+The motivation behind this is to be seen as extensions of the respective simulation code, and be easily discoverable in the context of that code. Looking at a package distribution repository, `fenicsprecice` should be close to `fenics`.
+
+The `adapter` is not part of the name, for concision reasons. The FEniCS community only wants to know that this package enables them to use preCICE with FEniCS. The "adapter" is preCICE-specific terminology.
```

Comment:
Minor rewording.

```suggestion
The term `adapter` is omitted for conciseness. The FEniCS community only wants to know that this package enables them to use preCICE with FEniCS. The "adapter" is preCICE-specific terminology.
```

Reply:
I think the "not part of the name" is the better description than "omitted". We still write sometimes the "`OpenFOAM-preCICE` adapter". "Adapter" is not part of the name, but part of the sentence. Similar to the "`ParticipantImpl` class".